fhttp/httprunner.go

```diff
@@ -69,8 +68,8 @@ func (httpstate *HTTPRunnerResults) Run(ctx context.Context, t periodic.ThreadID
 		httpstate.aborter.Abort(false)
 		log.Infof("Aborted run because of code %d - data %s", code, DebugSummary(body, 1024))
 	}
-	if code == http.StatusOK {
-		return true, "200"
+	if code >= 200 && code <= 299 {
```

Comment:
probably should use https://github.com/fortio/fortio/blob/6d264d850c06fd305c566afe545b1197b77900bb/fhttp/http_client.go#L999-L1001 ? (trying to remember why it isn't already)